scripts/test/Muon/loadrun_presenter_multiple_file_test.py

```diff
@@ -0,0 +1,221 @@
+import sys
+import six
+
+from Muon.GUI.MuonAnalysis.loadrun.load_run_model import LoadRunWidgetModel
+from Muon.GUI.MuonAnalysis.loadrun.load_run_view import LoadRunWidgetView
+from Muon.GUI.MuonAnalysis.loadrun.load_run_presenter import LoadRunWidgetPresenter
+
+from Muon.GUI.Common.muon_load_data import MuonLoadData
+
+import unittest
+
+if sys.version_info.major == 3:
+    from unittest import mock
+else:
+    import mock
+
+from PyQt4 import QtGui
+from PyQt4.QtGui import QApplication
+
+# global QApplication (get errors if > 1 instance in the code)
+QT_APP = QApplication([])
+
+
+class LoadRunWidgetIncrementDecrementMultipleFileModeTest(unittest.TestCase):
+    def run_test_with_and_without_threading(test_function):
+        def run_twice(self):
+            test_function(self)
+            self.setUp()
+            self.presenter._use_threading = False
+            test_function(self)
+
+        return run_twice
+
+    class Runner:
```

Comment:
is it worth putting runner in a shared script?

Reply:
It probably is, but may take a bit of time; I think the reason it appears in every tests class is due to its use of the global `qApp`.

Reply:
See comment below regarding the thread_model; this may actually fix some of the issues with the `Runner` and allow it to be gathered into a single place in the test file.